From 3b2d790a4bff5a15772576a8d1916b06f616e179 Mon Sep 17 00:00:00 2001
From: David Pacheco <dap@joyent.com>
Date: Wed, 16 Jan 2019 21:44:21 +0000
Subject: [PATCH] joyent/node-verror#65 tst.multierror.js failure on Node v8
 and later Reviewed by: Robert Mustacchi <rm@joyent.com> Reviewed by: Isaac
 Davis <isaac.davis@joyent.com> Approved by: Isaac Davis
 <isaac.davis@joyent.com>

---
 CHANGES.md             |  2 +-
 test/tst.multierror.js | 20 ++++++++++----------
 2 files changed, 11 insertions(+), 11 deletions(-)

diff --git a/CHANGES.md b/CHANGES.md
index bbb745a..5d2e0c7 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -2,7 +2,7 @@
 
 ## Not yet released
 
-None yet.
+* #65 tst.multierror.js failure on Node v8 and later
 
 ## v1.10.0
 
diff --git a/test/tst.multierror.js b/test/tst.multierror.js
index bc8bac7..a63125d 100644
--- a/test/tst.multierror.js
+++ b/test/tst.multierror.js
@@ -60,23 +60,23 @@ function main()
 	/* errorFromList */
 	mod_assert.throws(function () {
 		console.error(errorFromList());
-	}, /^AssertionError: errors \(\[object\]\) is required$/);
+	}, /^AssertionError.*: errors \(\[object\]\) is required$/);
 
 	mod_assert.throws(function () {
 		console.error(errorFromList(null));
-	}, /^AssertionError: errors \(\[object\]\) is required$/);
+	}, /^AssertionError.*: errors \(\[object\]\) is required$/);
 
 	mod_assert.throws(function () {
 		console.error(errorFromList({}));
-	}, /^AssertionError: errors \(\[object\]\) is required$/);
+	}, /^AssertionError.*: errors \(\[object\]\) is required$/);
 
 	mod_assert.throws(function () {
 		console.error(errorFromList('asdf'));
-	}, /^AssertionError: errors \(\[object\]\) is required$/);
+	}, /^AssertionError.*: errors \(\[object\]\) is required$/);
 
 	mod_assert.throws(function () {
 		console.error(errorFromList([ new Error(), 17 ]));
-	}, /^AssertionError: errors \(\[object\]\) is required$/);
+	}, /^AssertionError.*: errors \(\[object\]\) is required$/);
 
 	mod_assert.throws(function () {
 		console.error(errorFromList([ new Error(), {} ]));
@@ -95,23 +95,23 @@ function main()
 	/* errorForEach */
 	mod_assert.throws(function () {
 		console.error(errorForEach());
-	}, /^AssertionError: err must be an Error$/);
+	}, /^AssertionError.*: err must be an Error$/);
 
 	mod_assert.throws(function () {
 		console.error(errorForEach(null));
-	}, /^AssertionError: err must be an Error$/);
+	}, /^AssertionError.*: err must be an Error$/);
 
 	mod_assert.throws(function () {
 		console.error(errorForEach(err1));
-	}, /^AssertionError: func \(func\) is required$/);
+	}, /^AssertionError.*: func \(func\) is required$/);
 
 	mod_assert.throws(function () {
 		console.error(errorForEach(err1, {}));
-	}, /^AssertionError: func \(func\) is required$/);
+	}, /^AssertionError.*: func \(func\) is required$/);
 
 	mod_assert.throws(function () {
 		console.error(errorForEach({}, function () {}));
-	}, /^AssertionError: err must be an Error$/);
+	}, /^AssertionError.*: err must be an Error$/);
 
 	accum = [];
 	doAccum = function (e) { accum.push(e); };
-- 
2.21.0

